py3status/modules/whoami.py

```diff
@@ -7,6 +7,9 @@
         (default 1800)
     format: string to print (default '{username}')
 
+Format placeholders:
+    {username} effective userid
```

Comment:
`userid` can be misleading to the UID, which is a numerical value and not the username